schema-client-example/src/main/java/io/miragon/miranum/examples/schema/client/SchemaController.java

```diff
@@ -18,6 +18,6 @@ public class SchemaController {
 
     @GetMapping("/{ref}")
     public ResponseEntity<JsonNode> getSchema(@PathVariable final String ref) {
-        return ResponseEntity.ok(this.jsonApi.getSchema(ref).getSchema());
+        return ResponseEntity.ok(this.jsonApi.buildSchema(ref).getSchema());
```

Comment:
I think we should refactor this example. Would make more sense to have a running schema application, like in miranum platforms

Reply:
I totally agree with you 💯! 
I would handle this in a separate issue and PR. 
Hence I have created: https://github.com/Miragon/miranum-consulting/issues/26